src/modules/Writers.jl

```diff
@@ -157,4 +154,21 @@ dropheaders(h::Markdown.Header) = Markdown.Paragraph(Markdown.Bold(h.text))
 dropheaders(v::Vector) = map(dropheaders, v)
 dropheaders(other) = other
 
+
+# LaTeX Output.
+# -------------
+
+# TODO
+
+function render(io::IO, ::MIME"text/latex", node, page, doc)
```

Comment:
Hopefully not very relevant for long, but perhaps they should `error` at the moment?